src/connection.js

```diff
@@ -21,6 +21,8 @@ var Session = require('./session');
  * @extends TTransactionGateway
  */
 function Connection(id, options) {
+  Connection.validateOptions(options);
```

Comment:
Basically I vote for using options for params which are... optional :) Great for token, apisecret, but in case of address which is mandatory I would keep separate param.